src/actions/index.js

```diff
@@ -0,0 +1,38 @@
+import { newDataLoaded, drawerToggled, nameChanged, surnameChanged, mailChanged, passwordChanged } from "./creators";
+
+const loadData = (links) => (dispatch) => {
+    for (let i = 0; i < links.length; i++) {
```

Comment:
in your case you should use `Promise.all`. In this way you could guarantee that all data will be loaded, rather than just firing and forgetting about this requests.